spec/requests/v1/items_request_spec.rb

```diff
@@ -27,4 +27,23 @@
         expect(item.unit_price).to eq(42.00)
         expect(item.merchant_id).to eq(new_id)
     end
+
+    it "can update an existing Item" do
+        new_id = Merchant.create(name: "Joe").id
+        
+        item_id = Item.create({name: "JOY",
+        description: "To the World, my program works!",
+        unit_price: 42.00,
+        merchant_id: new_id}).id
+
+        previous_price = Item.last.unit_price
```

Comment:
variable containing soon-to-be overwritten quality for comparison in expect for assertion